docs/user-guide/configuration.md

```diff
@@ -68,6 +68,18 @@ pass a secondary option `"warn": true`.
 
 Different reporters may use these severity levels in different way, e.g. display them differently, or exit the process differently.
 
+##### Legacy Numbered Severities
+
+Before v3, configuration involved a system of numbered severities, like ESLint's.
+Numbered severities have been deprecated (and will be removed in v4), but can still be used for the time being —
+just expect a deprecation warning whenever stylelint runs.
+
+stylelint should be able to figure out whether you're using legacy numbered severities as opposed to the
+more recent standard. But there is one situation in which auto-detection won't work:
+if your config *only contains rules that take integers as their primary option* (e.g. indentation).
+**If this exception applies to you, you need to explicitly force legacy numbered severities by adding
+`legacyNumberedSeverities: true` to your config.**
```

Comment:
Wouldn't this be the opposite here, and if I only had `indentation` then I'd want to ensure I didn't use the legacy with `false`?